autorest/codegen/models/parameter.py

```diff
@@ -131,7 +131,7 @@ def _default_value(self) -> Tuple[Optional[Any], str, str]:
             default_value_declaration = "None"
         else:
             if isinstance(self.schema, ConstantSchema):
-                default_value = self.schema.constant_value
+                default_value = self.schema.get_declaration(self.schema.value)
```

Comment:
Originally I wanted to make a `default_value_declaration` for ConstantSchemas (the "default value" would be `self.value` instead of `self.default_value`), but I thought it would be too confusing to have a default value for a constant